learning/tour-of-beam/frontend/lib/models/unit_content.dart

```diff
@@ -27,13 +27,15 @@ class UnitContentModel {
   final String description;
   final String? taskSnippetId;
   final String? solutionSnippetId;
+  final List<String>? hints;
```

Comment:
What is the semantics difference between empty `hints` and null `hints`? Collections should be nullable only when there is a difference. Otherwise we got two states to check instead of one for the same purpose.